accessibility-checker-extension/src/ts/options/OptionsApp.tsx

```diff
@@ -429,12 +436,17 @@ export class OptionsApp extends React.Component<{}, OptionsAppState> {
                                 this.setState({ modalRuleSet: false });
                             }).bind(this)}
                         >
-                            <p style={{ maxWidth: "100%" }}><strong>Dated deployment: </strong> Use a rule set from a specific date
+                            <p style={{ maxWidth: "100%" }}><strong>Dated deployment</strong>: Use a rule set from a specific date
                                 for consistent testing throughout a project to replicate an earlier test</p>
 
-                            <p style={{ maxWidth: "100%" }}><strong>Preview rules: </strong> Try an experimental preview of possible future rule set</p>
+                            <p style={{ maxWidth: "100%" }}><strong>Preview rules</strong>: Try an experimental preview of a possible future rule set
+                                such as the new WCAG 2.2 related rules</p>
```

Comment:
```suggestion
                            <p style={{ maxWidth: "100%" }}><strong>Preview rules</strong>: Try an experimental preview of a possible future rule set.</p>
```
No reason to specifically mention WCAG 2.2 here since WCAG 2.2 is a guideline in the regular archives.